BLSQ/iaso
# POLIO-953: move rounds tabs to the left before scope

When people encode a campaign, they need to first access the Rounds tab to encode the dates, and then encode the corresponding scope(s). Though the Rounds tab was placed after Scope tab
![Capture d’écran 2023-04-12 à 12 41 11](https://user-images.githubusercontent.com/25134301/231434841-99e16eb1-5032-461b-aad5-68042cbaff5e.png)

Related JIRA tickets : [POLIO-953](https://bluesquare.atlassian.net/jira/software/c/projects/IA/boards/281?modal=detail&selectedIssue=POLIO-953&assignee=6231e24b50cceb00707a3be5)

## Self proofreading checklist



## How to test
- Go to Campaigns
- Create or Edit a campaign
- Rounds tab should be placed before Scope and Risk Assessment tab

## Print screen / video

![Capture d’écran 2023-04-12 à 12 40 55](https://user-images.githubusercontent.com/25134301/231434863-d0ad7de8-75c7-449c-94a3-dc0ce3827898.png)

[POLIO-953]: https://bluesquare.atlassian.net/browse/POLIO-953?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
